src/percy-node-client.js

```diff
@@ -93,6 +93,65 @@ const buildResourceUploadPromises = [];
 var registeredBreakpoints = {};
 
 
+/**
+ * Maximum trys to retrieve the build infomation from the Percy server.
+ * @const {number}
+ */
+let MAX_RETRIES_WHEN_PROCESSING = 1000;
+
+/**
+ * Sent another request to the Percy server if the number of tries does not exceed the limit. 
+ * @param {*} buildId Percy Build ID.
+ * @param {*} numRetries The number of get build requests to the server.
+ * @param {*} resolve Promise resolve function.
+ */
+function retry(buildId, numRetries, resolve) {
+  if (numRetries < MAX_RETRIES_WHEN_PROCESSING) {
+      // Retry with recursion with retries incremented
+      return setTimeout(getBuild, MAX_RETRIES_WHEN_PROCESSING, buildId, numRetries + 1, resolve);
+  } else {
+      console.error('Retries exceeded. Exiting.');
+  }
+}
+
+/**
+ * Retrieve the build information from the Percy server.
```

Comment:
make this description more descriptive of what the function is doing. e.g. checking for diffs, displaying an error if there are diffs.